### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by dawmster**
Polish translation - pl.yml 

utf8 encoded

Cheers

### CODE REVIEW

**Comment by dawmster** [comment]
Also as a gist here: https://gist.github.com/dawmster/9a32ea539e1a50a0dc6d5706d95893e6

File: translations/pl.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+en:
```
Comment (zquestz) [inline translations/pl.yml]: This key should be `pl:`

**Comment by zquestz** [comment]
I will fix the broken key. Thanks a ton for helping out!

**Comment by dawmster** [comment]
My pleasure, cheers.

**Pull request merged**